imagebuildah/build.go

```diff
@@ -211,7 +211,12 @@ func BuildDockerfiles(ctx context.Context, store storage.Store, options define.B
 	}
 
 	if options.AllPlatforms {
-		options.Platforms, err = platformsForBaseImages(ctx, logger, paths, files, options.From, options.Args, options.SystemContext)
+		additionalBuildContexts := options.AdditionalBuildContexts
+		if additionalBuildContexts == nil {
+			// Create empty map for additionalbuildContexts
+			additionalBuildContexts = make(map[string]*define.AdditionalBuildContext)
+		}
```

Comment:
For this case you're going to want to overwrite `options.AdditionalBuildContexts` so that the various `platformOptions` structures created below don't see `nil` values for that field.